app/models/barcode.rb

```diff
@@ -39,14 +39,16 @@ class Barcode < ApplicationRecord
     heron_tailed: 18,
     randox: 19,
     uk_biocentre_v4: 20,
-    cambridge_a_z_v2: 21
+    cambridge_a_z_v2: 21,
```

Comment:
All looks fine.
My question: 
why separate out the specific EGG and EGC from the general Eagle one?
e.g. Milton Keynes and the others are not done, so if they start appearing you'd need another change.

Reply:
As discussed - I'll put it here for record - it seemed easier to have a barcode just fall into one category. There is code that says "Here's a barcode, what category am I in?" and I think it will be easier for future dev to have just one answer to that and not have to guess which one it pulled back. Think the categories are used to enforce uniqueness within the categories.